index.html

```diff
@@ -20,54 +20,68 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
+    <div class="menu-bar hidden-menu">
```

Comment:
- [ ] Could you fix the horizontal overflowing issue? Modern web pages are expected to be fully responsive because of the high volume of mobile users on the Internet. 👍

![image](https://user-images.githubusercontent.com/63473584/215590694-20787edb-afae-476b-934b-ea112e6b507f.png)